executor.go

```diff
@@ -47,26 +43,21 @@ type Executor struct {
 	Host    string
 	Cluster *Cluster
 
-	// Client used for remote HTTP requests.
-	HTTPClient *http.Client
+	// Client used for remote requests.
+	client InternalClient
 
 	// Maximum number of SetBit() or ClearBit() commands per request.
 	MaxWritesPerRequest int
 }
 
 // NewExecutor returns a new instance of Executor.
-func NewExecutor(clientOptions *ClientOptions) *Executor {
+func NewExecutor(clientOptions *ClientOptions) (*Executor, error) {
```

Comment:
why did you add an `error` to the return here?

Reply:
I think I called a function which returned an `error` there and removed that code without without the `error`. Will remove it.